app/admin/governance/credential_definitions.py

```diff
@@ -22,6 +22,18 @@ async def create_credential_definition(
     credential_definition: CredentialDefinition,
     aries_controller: AriesAgentControllerBase = Depends(agent_selector),
 ):
+    """
+    Create a credential definition.
+
+    Parameters:
+    -----------
+    credential_definition: CredentialDefinition
+        Payload for creating a credential definition.
+
+    Returns:
+    --------
+    The response object obtained from creating a credential definition.
```

Comment:
The following applies to much of the comments below also:

Do we want to either at least give a pydantic model for return type (when python standard type is not applicable) or do we possibly even want to provide n example of the return objects structure). For example, should we return an invitation should we ensure we have an invitation pydantic model (or other custom type for that) we reference here and a developer can look up or should be reference it it here AND even provide a a description of the strcuture/fields?